Tools/MonoGame.Content.Builder.Editor/Common/PipelineSettings.cs

```diff
@@ -19,7 +19,9 @@ public class PipelineSettings
 
         public List<string> ProjectHistory;
         public string StartupProject;
-        public Microsoft.Xna.Framework.Point Size;
+        public System.Drawing.Size WindowSize;
```

Comment:
@harry-cpp - are we using `System.Drawing` in other places... is it portable for us?

Reply:
Its not, it should not be used.

Reply:
Ok, gonna change it.